src/documents/index.py

```diff
@@ -254,6 +268,12 @@ def _get_query_filter(self):
         else:
             return query.Or(user_criterias) if len(user_criterias) > 0 else None
 
+    def evalBoolean(self, val):
```

Comment:
I think this could be improved, just a little.  If the value happens to ever include upper case, it would end up with None

```python
val.lower() in {"true", "1"}
```

Reply:
awesome thanks

Reply:
Actually, the whole thing could be replaced with the above, unless None is a valid response when the value isn't a true or false (which I assume it would be always valid)

Reply:
Ah thanks, yea I thought so and changed it which caused a failure and I quickly changed but actually this helped discover an issue, should be sorted. Thanks!